tests/python/unittest/test_operator.py

```diff
@@ -4417,7 +4417,12 @@ def test_pick_helper(index_type=np.int32):
             sshape = bshape.copy()
             sshape[axis] = 1
             data = np.random.uniform(-1, 1, size=bshape)
-            index = np.random.randint(0, bshape[axis], size=sshape)
+            mode = np.random.choice(a=['clip', 'wrap']) 
```

Comment:
Please test both choices for `mode` using a for-loop instead of randomized testing for `mode`